package.json

```diff
@@ -151,7 +151,7 @@
     "redux-logger": "^2.0.2",
     "redux-thunk": "^0.1.0",
     "semver-regex": "^1.0.0",
-    "sequelize": "^4.37.0",
+    "sequelize": "^4.37.10",
```

Comment:
must update package-lock when changing this

Reply:
Was there a reason you changed this?

Reply:
must have updated automatically when I git pulled or ran npm install. I'll change it back

Reply:
If you run "npm install sequelize" it will install the latest sequelize version and update package.json. Generally you should just run "npm install", and it will install the latest version for all packages without changing package.json.

And generally, package-lock.json will be updated any time you run "npm install", but you should only check it in when necessary (new package or major version update).

It doesn't really matter, functionally it's the same, but I think it's better to keep the PRs clean and only check in necessary changes.

I think you should revert the changes to package-lock.json as well.

@iamigo make sense?